src/main/scala/firrtl/passes/AnnotateMemMacros.scala

```diff
@@ -21,35 +21,35 @@ case class AppendableInfo(fields: Map[String, Any]) extends Info {
 }
 
 object AnalysisUtils {
-
-  def getConnects(m: Module) = {
-    val connects = mutable.HashMap[String, Expression]()
-    def getConnects(s: Statement): Statement = {
-      s map getConnects match {
+  type Connects = collection.mutable.HashMap[String, Expression]
```

Comment:
I'm not really sure how this works, but the reason I wanted to do a "connects.toMap" at the end was to have getConnects return an immutable map even if, inside the function, you're appending to a mutable HashMap. Is, then, returning a "Connects" type = Mutable Hashmap of [String, Expression] bad? 

Not really sure how that works...